components/higher-order/with-api-data/index.js

```diff
@@ -18,32 +18,26 @@ export default ( mapPropsToData ) => createHigherOrderComponent( ( WrappedCompon
 	class APIDataComponent extends Component {
```

Comment:
Should we also add a note that `withApiData` is planned to be deprecated in on of the future releases and `withSelect` is the recommended approach?

Reply:
Are there any challenges? It seems like a good task to attack during Contributor Day this week 😃